app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = False
+        return self.appetite
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            is_hungry: bool = True,
+            appetite: int = 3
+    ) -> None:
+        super().__init__(name, appetite, is_hungry)
```

Comment:
> Write Cat class which is a child of Animal. It should have the __init__ method with two arguments:
> 
> name - the name of a cat
> is_hungry - with True default value

It accepts two params, not 3. Then you can use:
super().__init__(name, 3, is_hungry)
Fix the same with Dog